tests/Oryx.Tests.Common/TelemetryClientHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Oryx.Tests.Common;
+using System;
+
+public static class TelemetryClientHelper
+{
+    public static string aiKey = TestConstants.AiKey;
+    public static TelemetryClient GetTelemetryClient()
+    {
+        var telemetryConfig = new TelemetryConfiguration();
+        if (!string.IsNullOrWhiteSpace(aiKey))
+        {
+            telemetryConfig.ConnectionString = aiKey;
+        }
+        return new TelemetryClient(telemetryConfig);                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
+    }
```

Comment:
I think we should make this method a little more robust when it comes to where the connection string comes from -- specifically, it would be great if we could first check to see if an environment variable was set that we could use, and then fall back to `TestConstants.AiKey` if it wasn't set.

You'd want to add another value to `TestConstants` that is the environment variable we're reading, something like 

```cs
public const string AppInsightsConnectionStringEnvironmentVariable = "TEST_AI_CONNECTION_STRING";
```

Then we could have this method work as so:

```cs
public static TelemetryClient GetTelemetryClient()
{
    var connectionString = Environment.GetEnvironmentVariable(TestConstants.AppInsightsConnectionStringEnvironmentVariable)
        ?? TestConstants.AppInsightsConnectionString;
    var telemetryConfig = new TelemetryConfig()
    {
        ConnectionString = connectionString,
    };
    return new TelemetryClient(telemetryConfig);
}
```

Reply:
Okay. That makes sense.